SmartPlaces.Facilities/lib/IngestionManager.Mapped/src/MappedGraphManager.cs

```diff
@@ -81,7 +81,7 @@ public MappedGraphManager(ILogger<MappedGraphManager> logger, IHttpClientFactory
         /// Creates a query that returns all the sites associated to an organization.
         /// </summary>
         /// <returns>A formatted graph query.</returns>
-        public string GetOrganizationQuery()
+        public virtual string GetOrganizationQuery()
```

Comment:
Why are all these chaning to virtual?

Reply:
I contemplated building a version of the class that is specific to the Mapped-Willow integration. It also allows me to create a class that overrides them, and I can test with new forms of the query without having to check in changes.